src/web_interface/templates/home.html

```diff
@@ -62,20 +86,22 @@ <h4 class="mb-3">Latest Firmware Submissions</h4>
 
     <div class="row justify-content-center">
     	{% if (latest_comments | length) > 0 %}
-            <div class="p-2 mt-3" style="width: 705px;">
+            <div class="p-2 mt-3" style ="width: 705px;">
                 <h4 class="mb-3">Latest Comments</h4>
                 <ul class="list-group">
                     {% for comment in latest_comments %}
                         <li class="list-group-item p-2 clickable li-hover" onclick="location.href='/analysis/{{ comment.uid }}'">
-                            <a href="/analysis/{{ comment.uid }}">
+                            <a href="/analysis/{{ comment.uid }}" style="color: #515151;">
                                 <div class="p-1" style="width: 100%; height: 100%; font-size: 14px;">
-                                    <span style="font-family: monospace; color: #515151; font-size: 14px; word-break: break-word;">
+                                    <div class="bubble bubble-triangle">
+<!--                                    <span style="font-family: monospace; font-size: 14px; word-break: break-word;">-->
                                         {{ comment.comment | safe }}
-                                    </span>
+<!--                                    </span>-->
+                                    </div>
                                     <br>
```

Comment:
<br /> is nineties html. Use classes `mb-1` to `mb-3` instead

Reply:
fixed